app/models/role.rb

```diff
@@ -30,6 +30,12 @@ def organisations
     links["ordered_parent_organisations"]
   end
 
+  def currently_occupied?
```

Comment:
This method is very similar to the one below is, `current_holder`. To avoid duplicate code, you could write this method as:

```ruby
def currently_occupied?
  current_holder.present?
end
```

But then I wonder whether this method has much benefit, and in your view, you could just do:

```
role.current_holder ? { ... } : nil
```

Reply:
Okay definitely an oversight on my part, it kind of doesn't have much actual benefit, but using a methods like that keeps it consistent with similar views across Collections, which I personally think is a good thing. You may disagree though